src/CommunityToolkit.Maui/Behaviors/Validators/EmailValidationBehavior.shared.cs

```diff
@@ -117,4 +115,13 @@ static string DomainMapper(Match match)
 			return match.Groups[1].Value + domainName;
 		}
 	}
+
+	[GeneratedRegex(@"^([1-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])(\.([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])){3}$", RegexOptions.None, 250)]
+	private static partial Regex ValidIpv4Regex();
```

Comment:
We don't use private modifiers explicitly, is it required?

Reply:
I would say this (and others Regex methods) should be `protected` since this class can be `inherited`

Reply:
> We don't use private modifiers explicitly, is it required?

Yup! For some reason, partial methods explicitly require modifiers even if the default is private. 🤷‍♂️

Reply:
If I'm not wrong the only one that you don't need to add the access modifiers is when the return type is `void`. If I remember well, the lang. the team did it to not break old partial methods, so if your method returns something other than `void` the modifiers are required.

Reply:
> should be protected since this class can be inherited
Sure! I'm not against it.

The current `Regex`s are `private`, not `protected`. So this wouldn't be a breaking change, but we would want to add unit tests around the newly `protected Regex` methods to ensure that we don't accidentally break anything in future PRs.

Reply:
Since the user can extend from `EmailValidationBehavior` I can see value in changing those `Regex` methods to be `protected`, that way the user can reuse then to perform more checks... Otherwise, they can't reuse anything from the base class.

Also, I'm fine if you aren't comfortable to do this right now, and we can wait someone from the community to request this change.

Reply:
Let's merge this PR and then I'll open a new PR to promote the `[GeneratedRegex]` methods to protected

Reply:
> Let's merge this PR and then I'll open a new PR to promote the `[GeneratedRegex]` methods to protected

Here's the branch where I've begun the work: https://github.com/CommunityToolkit/Maui/tree/Promote-%60-GeneratedRegex%5D%60-Methods-to-%60protected%60